README.md

```diff
@@ -110,8 +112,40 @@ The last published version of the package become [available for download](htt
 
 ### Examples
 
-> **Will come soon**
+```js
+var fs = require('fs');
+var promisify = require('promisify-es6')
```

Comment:
The variable `promisify` does not seem to be used anywhere below in your examples.

(Should you decide to delete this line?)